plugins/kubernetes-backend/api-report.md

```diff
@@ -234,7 +248,7 @@ export type KubernetesObjectTypes =
 // @public (undocumented)
 export interface KubernetesServiceLocator {
   // (undocumented)
-  getClustersByServiceId(serviceId: string): Promise<ClusterDetails[]>;
+  getClustersByEntity(entity: Entity): Promise<ClusterDetails[]>;
```

Comment:
Probably should also wrap up the response here in an object, something like `Promise<{ clusters: ClusterDetails[]}>` so we can grow the API further in the future without breaking the return type too much.

Reply:
Makes sense!